recipes/metpy/meta.yaml

```diff
@@ -0,0 +1,41 @@
+package:
+  name: metpy
+  version: !!str 0.3.0
```

Comment:
> I don't like the jinja variable when the variable will be used only once

I agree. Though it is hard to come up with a useful example which doesn't have real source data! :smile: